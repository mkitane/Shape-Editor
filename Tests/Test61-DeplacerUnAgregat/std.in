C cercle 1 1 2
C cercle1 2 2 3
OA agregat cercle cercle1
LIST
MOVE agregat 10 10
LIST
EXIT
