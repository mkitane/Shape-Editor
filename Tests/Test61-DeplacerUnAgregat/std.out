OK
OK
OK
OA agregat cercle cercle1 
C cercle 1 1 2
C cercle1 2 2 3
OK
OA agregat cercle cercle1 
C cercle 11 11 2
C cercle1 12 12 3
